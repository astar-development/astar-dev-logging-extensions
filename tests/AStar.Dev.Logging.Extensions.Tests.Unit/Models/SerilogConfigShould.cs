using AStar.Dev.Utilities;

namespace AStar.Dev.Logging.Extensions.Models;

public sealed class SerilogConfigShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new SerilogConfig().ToJson().ShouldMatchApproved();
}