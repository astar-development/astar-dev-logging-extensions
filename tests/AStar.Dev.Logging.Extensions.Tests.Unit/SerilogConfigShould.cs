using AStar.Dev.Logging.Extensions.Models;
using AStar.Dev.Utilities;

namespace AStar.Dev.Logging.Extensions;

public sealed class SerilogConfigShould
{
    [Fact]
    public void ContainTheExpectedProperties() =>
        new SerilogConfig().ToJson().ShouldMatchApproved();
}
