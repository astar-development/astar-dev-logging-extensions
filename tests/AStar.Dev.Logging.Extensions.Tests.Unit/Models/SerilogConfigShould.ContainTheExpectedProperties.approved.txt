{
  "serilog": {
    "enrich": [],
    "writeTo": [],
    "minimumLevel": {
      "default": "Verbose",
      "override": {
        "microsoftAspNetCore": "",
        "systemNetHttp": "",
        "aStar": ""
      }
    }
  },
  "logging": {
    "console": {
      "formatterName": "",
      "formatterOptions": {
        "singleLine": false,
        "includeScopes": false,
        "timestampFormat": "HH:mm:ss ",
        "useUtcTimestamp": true,
        "jsonWriterOptions": {
          "indented": false
        }
      }
    },
    "applicationInsights": {
      "logLevel": {
        "default": "",
        "microsoftAspNetCore": "",
        "aStar": ""
      }
    }
  }
}